zubhub_backend/zubhub/activities/admin.py

```diff
@@ -1,3 +1,49 @@
 from django.contrib import admin
+from django.contrib import messages
 
-# Register your models here.
+from .models import Activity
+
+
+def make_unpublished(modeladmin, request, queryset):
+    """
+    This function is used to unpublish selected activities
+    """
+    queryset.update(publish=False)
+    messages.success(request, "Selected records were unpublished successfully.")
+
+
+make_unpublished.short_description = "Unpublish selected activities"
+
+
+def make_published(modeladmin, request, queryset):
+    """
+    This function is used to publish selected activities
+    """
+    queryset = queryset.filter(publish=False)
+    queryset.update(publish=True)
+    messages.success(request, "Selected records were published successfully.")
+
+
+make_published.short_description = "Publish selected activities"
+
+
+def delete_selected(modeladmin, request, queryset):
```

Comment:
a bit confused. Doesn't django admin do this automatically by itself already?

Reply:
Not really, it doesn't do this. Reason why I added custom actions